core/arch/arm/plat-imx/gpcv2.c

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
This actually is not a driver, no driver framework. This is just to handle gpc registers and directly provide functions for imx psci code. I prefer keep it in plat-imx.